lib/cpp/libthriftQt.pro

```diff
@@ -0,0 +1,161 @@
+    src/thrift/concurrency/Util.h \
+    src/thrift/async/TAsyncChannel.h \
+    src/thrift/async/TConcurrentClientSyncInfo.h \
+    src/thrift/protocol/TCompactProtocol.h
+
+
+win32 {
+    SOURCES += \
+    src/thrift/async/TAsyncChannel.cpp \
+    src/thrift/concurrency/BoostMonitor.cpp \
+    src/thrift/concurrency/BoostMutex.cpp \
+    src/thrift/concurrency/BoostThreadFactory.cpp \
+    src/thrift/concurrency/ThreadManager.cpp \
+    src/thrift/concurrency/TimerManager.cpp \
+    src/thrift/server/TServerFramework.cpp \
+    src/thrift/server/TSimpleServer.cpp \
+    src/thrift/server/TThreadedServer.cpp \
+    src/thrift/server/TThreadPoolServer.cpp \
+    src/thrift/windows/GetTimeOfDay.cpp \
+    src/thrift/windows/OverlappedSubmissionThread.cpp \
+    src/thrift/windows/SocketPair.cpp \
+    src/thrift/windows/TWinsockSingleton.cpp \
+    src/thrift/windows/WinFcntl.cpp \
+    src/thrift/transport/TFileTransport.cpp \
+    src/thrift/transport/THttpClient.cpp \
+    src/thrift/transport/THttpServer.cpp \
+    src/thrift/transport/THttpTransport.cpp \
+    src/thrift/transport/TPipe.cpp \
+    src/thrift/transport/TPipeServer.cpp \
+    src/thrift/transport/TServerSocket.cpp \
+    src/thrift/transport/TSimpleFileTransport.cpp \
+    src/thrift/transport/TSocket.cpp \
+    src/thrift/transport/TSSLSocket.cpp
+
+    HEADERS  += \
+    src/thrift/async/TAsyncChannel.h \
+    src/thrift/concurrency/BoostThreadFactory.h \
+    src/thrift/server/TServerFramework.h \
+    src/thrift/server/TSimpleServer.h \
+    src/thrift/server/TThreadedServer.h \
+    src/thrift/server/TThreadPoolServer.h \
+    src/thrift/windows/config.h \
+    src/thrift/windows/GetTimeOfDay.h \
+    src/thrift/windows/Operators.h \
+    src/thrift/windows/OverlappedSubmissionThread.h \
+    src/thrift/windows/SocketPair.h \
+    src/thrift/windows/TWinsockSingleton.h \
+    src/thrift/windows/WinFcntl.h \
+    src/thrift/transport/TFileTransport.h \
+    src/thrift/transport/THttpClient.h \
+    src/thrift/transport/THttpServer.h \
+    src/thrift/transport/THttpTransport.h \
+    src/thrift/transport/TPipe.h \
+    src/thrift/transport/TPipeServer.h \
+    src/thrift/transport/TServerSocket.h \
+    src/thrift/transport/TServerTransport.h \
+    src/thrift/transport/TSimpleFileTransport.h \
+    src/thrift/transport/TSocket.h \
+    src/thrift/transport/TSSLSocket.h
+
+    INCLUDEPATH += "D:\libs\OpenSSL-Win64_1.0.1i\include"
+
+    LIBS += -LD:\libs\boost_1_58_0_X64_VS2013\lib64-msvc-12.0 -llibboost_thread-vc120-mt-gd-1_58 -llibboost_chrono-vc120-mt-gd-1_58
+    LIBS += -LD:\libs\OpenSSL-Win64_1.0.1i\lib\VC -llibeay32MDd -lssleay32MTd
+
+    CONFIG(debug, debug|release) {
+        DESTDIR = ../cpp/DebugQtWin
+        OBJECTS_DIR = ../cpp/DebugQtWin/.obj
+    } else {
+        DESTDIR = ../cpp/releaseQtWin
+        OBJECTS_DIR = ../cpp/releaseQtWin/.obj
+    }
+}
+android {
+    #SOURCES +=
+    HEADERS += libthriftQt/thrift/config.h \
```

Comment:
please use `#include <thrift/thrift-config.h>`
`config.h` is generated and should not be part of the source tree